Dockerfile

```diff
@@ -1,72 +1,68 @@
-FROM python:3.9-slim
+# We use Debian images because they are considered more stable than the alpine
+# ones becase they use a different C compiler. Debian images also come with
+# all useful packages required for image manipulation out of the box. They
+# however weigh a lot, approx. up to 1.5GiB per built image.
+FROM python:3.9 as production
 
-# Install packages needed to run your application (not build deps):
-# We need to recreate the /usr/share/man/man{1..8} directories first because
-# they were clobbered by a parent image.
-RUN set -ex \
-    && RUN_DEPS=" \
-        libexpat1 \
-        libjpeg62-turbo \
-        libpcre3 \
-        libpq5 \
-        mime-support \
-        postgresql-client \
-        procps \
-        zlib1g \
-    " \
-    && seq 1 8 | xargs -I{} mkdir -p /usr/share/man/man{} \
-    && apt-get update && apt-get install -y --no-install-recommends $RUN_DEPS \
-    && rm -rf /var/lib/apt/lists/*
+# Install dependencies in a virtualenv
+ENV VIRTUAL_ENV=/venv
```

Comment:
Do we need a virtual env?

Reply:
As we're not using poetry here, technically no, but it makes the environment more like our other production applications

Reply:
Not a huge deal, just thinking that this is likely going to be used as a 'best practices' `Dockerfile` example for a lot of people starting out with Wagtail, and this adds a layer of complexity without adding much value.